controller/deploy.go

```diff
@@ -60,26 +64,51 @@ func (d Deploy) buildConfigExists(name string) (bool, error) {
 }
 
 func (d Deploy) create(dc DeployCmd) (*DeployResponse, error) {
-	labels := map[string]string{
-		"rmmap/guid":    dc.CloudAppGUID(),
-		"rhmap/project": dc.Project(),
-		"rhmap/domain":  dc.DomainName(),
+	dc.SetAppTag("redis")
+	d.createRedis(dc)
+	dc.AddEnvVar("REDIS_HOST", dc.AppName())
+	dc.SetAppTag("cloud")
+	return d.createCloudApp(dc)
+}
+
+func (d Deploy) createRedis(dc DeployCmd) error {
+	if err := d.paaSService.CreateService(dc, "redis app", 8001); err != nil {
+		return err
+	}
+	if err := d.paaSService.CreateDeploymentConfig(dc, "docker.io/rhmap/redis:2.18.22", "redis deployment config"); err != nil {
```

Comment:
bah, yes this should be in the config, that comment is intended for the node From image.